server/embed/config.go

```diff
@@ -755,8 +764,9 @@ func (cfg *Config) AddFlags(fs *flag.FlagSet) {
 	// experimental
 	fs.BoolVar(&cfg.ExperimentalInitialCorruptCheck, "experimental-initial-corrupt-check", cfg.ExperimentalInitialCorruptCheck, "Enable to check data corruption before serving any client/peer traffic.")
 	fs.DurationVar(&cfg.ExperimentalCorruptCheckTime, "experimental-corrupt-check-time", cfg.ExperimentalCorruptCheckTime, "Duration of time between cluster corruption check passes.")
-	fs.BoolVar(&cfg.ExperimentalCompactHashCheckEnabled, "experimental-compact-hash-check-enabled", cfg.ExperimentalCompactHashCheckEnabled, "Enable leader to periodically check followers compaction hashes.")
-	fs.DurationVar(&cfg.ExperimentalCompactHashCheckTime, "experimental-compact-hash-check-time", cfg.ExperimentalCompactHashCheckTime, "Duration of time between leader checks followers compaction hashes.")
+	fs.BoolVar(&cfg.ExperimentalCompactHashCheckEnabled, "experimental-compact-hash-check-enabled", cfg.ExperimentalCompactHashCheckEnabled, "Enable leader to periodically check followers compaction hashes. Deprecated, and will be decommissioned in v3.7. Use '--feature-gates=CompactHashCheck=true' instead")
```

Comment:
@ahrtr added a new deprecation warnings for v3.7. Would be good to register "experimental-compact-hash-check-enabled" and "experimental-compact-hash-check-time" in  [deprecatedFlags](https://github.com/etcd-io/etcd/blob/026bbecd25e0f9312b99175940a6761580cf5643/server/etcdmain/config.go#L60-L65). Please take a look 
https://github.com/etcd-io/etcd/pull/18998

Reply:
+1